core/iwasm/common/wasm_runtime_common.c

```diff
@@ -4212,6 +4212,9 @@ static V128FuncPtr invokeNative_V128 = (V128FuncPtr)(uintptr_t)invokeNative;
           || defined(BUILD_TARGET_RISCV64_LP64) */
 #endif /* end of defined(_WIN32) || defined(_WIN32_) */
 
+#if defined(__GNUC__)
+__attribute__((no_sanitize_address))
```

Comment:
Not sure why add `no_sanitize_address` here? Maybe we can resolve the issue, do you know how to reproduce the issue?

Reply:
Got it, thanks.